githooks/pre-commit-adorno

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+foundryup
```

Comment:
I don't think I want tihs hook to automatically update the tooling on my machine.

@gravityblast wdyt?

Reply:
I think the same @0x-r4bbit . I don't really like precommits forced in the repo. 
I prefer them to be forced on CI but optional in local